build.gradle

```diff
@@ -13,19 +13,19 @@ buildscript {
 
 allprojects {
     repositories {
-        jcenter()
+        mavenCentral()
         google()
     }
 }
 
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 29
-    buildToolsVersion '29.0.3'
+    compileSdkVersion 31
+    buildToolsVersion '30.0.3'
```

Comment:
we can just take out the buildToolsVersion now  https://stackoverflow.com/questions/24521017/android-gradle-buildtoolsversion-vs-compilesdkversion

Reply:
buildToolsVersion '30.0.3' line removed. Thank you!